migrations/2023-09-28-084231_import_user_settings_rate_limit/up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE local_site_rate_limit
+    ADD COLUMN import_user_settings int NOT NULL DEFAULT 1;
+
+ALTER TABLE local_site_rate_limit
+    ADD COLUMN import_user_settings_per_second int NOT NULL DEFAULT 86400;
```

Comment:
i'm a bit confused by this naming. it sounds like it allows 86400 imports per second but what it really means is it allows only one import per 86400 seconds. so maybe it should be called smth like `import_user_settings_rate_limit_seconds`?

Reply:
Yes all the rate limits have these confusing names so at least its consistent. It would be nice to rename them in the future to be clearer.